### PR TITLE
Navbar, html, css and links

### PR DESCRIPTION
**Pull request by raffoz**

### CODE REVIEW

File: app/views/shared/_navbar.html.erb [hunk 1]
```diff
@@ -13,33 +13,33 @@
 
       <% if user_signed_in? && current_user.role == "owner" %>
 
-        <li class="nav-item active">
+<!--         <li class="nav-item active">
```
Comment (kleyv) [inline app/views/shared/_navbar.html.erb:16]: Do you want to keep this or did you forget to delete it?

File: app/views/shared/_navbar.html.erb [hunk 2]
```diff
@@ -13,33 +13,33 @@
...
           <%= link_to "All projects", projects_path, class: "nav-link" %>
         </li>
         <li class="nav-item">
           <%= link_to "My projects", personal_projects_path, class: "nav-link" %>
-        </li>
+        </li> -->
```
Comment (kleyv) [inline app/views/shared/_navbar.html.erb:21]: Do you want to keep this or did you forget to delete it?

File: app/views/shared/_navbar.html.erb [hunk 3]
```diff
@@ -13,33 +13,33 @@
...
+            <%= link_to "My projects", personal_projects_path, class: "dropdown-item" %>
+            <%= link_to "Create a new project", new_project_path, class: "dropdown-item" %>
             <%= link_to "Log out", destroy_user_session_path, method: :delete, class: "dropdown-item" %>
           </div>
         </li>
 
       <% elsif user_signed_in? && current_user.role == "investor" %>
 
-        <li class="nav-item active">
+<!--         <li class="nav-item active">
```
Comment (kleyv) [inline app/views/shared/_navbar.html.erb:35]: Do you want to keep this or did you forget to delete it?

File: app/views/shared/_navbar.html.erb (hunk 1 above)
Comment (raffoz) [reply app/views/shared/_navbar.html.erb:16]: Keep

File: app/views/shared/_navbar.html.erb (hunk 2 above)
Comment (raffoz) [reply app/views/shared/_navbar.html.erb:21]: Keep

File: app/views/shared/_navbar.html.erb (hunk 3 above)
Comment (raffoz) [reply app/views/shared/_navbar.html.erb:35]: Keep

**Pull request merged**